examples/new_design_examples/new_design_tf_discs.m

```diff
@@ -25,46 +25,39 @@
 
 % This file is part of NOSNOC.
 
-%
-%
-
 %% Manipulation of two discs
-
-%%
-clear all;
-close all;
-clc;
+clear; close all; clc;
 import casadi.*
-
+import nosnoc.*
 %%
 filename = 'discs_manipulation.gif';
 %% discretizatioon
-N_stg = 10; % control intervals
-N_FE = 4;  % integration steps per control interval
+N_stg = 15; % control intervals
+N_FE = 2;  % integration steps per control interval
 T = 2;
 %% init
 problem_options = nosnoc.Options();
 solver_options = nosnoc.solver.Options();
 model = nosnoc.model.Cls();
-% settings
+%% settings
 problem_options.rk_scheme = RKSchemes.RADAU_IIA;
-problem_options.n_s = 2;  % number of stages in IRK methods
-problem_options.time_freezing = 1;
-problem_options.cross_comp_mode = 7;
+problem_options.n_s = 2;  % number of stages in IRK methods (TODO@Anton, other variations, e.g. Stewart, with n_s>1 converge nicely, this setting very slow)
```

Comment:
as indicated in the comment, heaviside and costum lifted converges much slower than stewart or heaviside without costum lift for all other settings the same. initialization of lifting variables is the problem?

Reply:
I believe this is now fixed with changed initialization and adding initialization calculation for the lifting variables.